frontend/src/components/viewers/Tensorboard.test.tsx

```diff
@@ -101,4 +108,35 @@ describe('Tensorboard', () => {
   it('is aggregatable', () => {
     expect(TensorboardViewer.prototype.isAggregatable()).toBeTruthy();
   });
+
+  it('set state', async () => {
```

Comment:
Recommend reading this: https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change
Main point: test UI behavior, not implementation details.
Although the article advertises react testing library, but you can do similar testing style with enzyme.

Steps for this test:
1. mount the component (not shallow)
2. simulate clicking the select input
3. simulate clicking an item
4. verify UI shows your chosen item
5. click start tensorboard button, verify it called api with the correct values

In this way, no matter how you refactor this UI component, you test remains the same as long as user behavior is the same.

Reply:
(please address behavior comments first, before you actually work on the test)